SAMtags.tex

```diff
@@ -234,13 +239,39 @@ \subsection{Barcodes}
 
 \begin{description}
 \item[BC:Z:\tagvalue{sequence}]
-Barcode sequence, with any quality scores stored in the {\tt QT} tag.
-
-\item[QT:Z:\tagvalue{qualities}]
-Phred quality of the barcode sequence in the {\tt BC} (or {\tt RT}) tag. Same encoding as {\sf QUAL}.
+Barcode sequence (Identifying the sample/library), with any quality scores (optionally) stored in the {\tt QT} tag.
+The {\tt BC} tag should match the {\tt QT} tag in length. 
+In the case of multiple unique molecular identifiers (e.g., one on each end of the template) the recommended implementation concatenates all the barcodes and qualities and places a dash (`{\tt -}') between the barcodes from the same template. 
```

Comment:
@nh13 I was thinking between barcodes, but I have little experience with more than one barcode, so that's just mirroring how I implemented it in Picard. if you have a different suggestion, please let me know. 

Regarding concatenating qualities. @nh13 what is your opinion on this? I think for parsing it is easier to have a separator in the quality field. but I am reluctant to use a `-` since that's a valid quality character. We could use a character that is not legal for quality (for example a space ` `) that will make parsing a little simpler. 

Regarding length vis-a-vis indels. @jkbonfield I'm not sure I understand how indel have anything to do with it. Let's assume the barcode is truly AACG and the sequencer read AAGT (skipping the C and reading into the next nucleotide.) and assume further that the correcting algorithm is smart enough to figure this out, it emit AACG as the corrected barcode....still the same length.